ibtk/include/ibtk/EdgeDataSynchronization.h

```diff
@@ -69,7 +69,8 @@ class EdgeDataSynchronization : public SAMRAI::tbox::DescribedClass
         /*!
          * \brief Default constructor.
          */
-        inline SynchronizationTransactionComponent(int data_idx = -1, const std::string& coarsen_op_name = "NONE")
+        inline SynchronizationTransactionComponent(int data_idx = invalid_index,
```

Comment:
I think some other stuff got tangled up in here.

Reply:
Looks like I messed up rebasing against `master`. That is what I get for force pushing code while adjudicating a wrestling match between my kids. I think this is fixed now.